common/tools/dev-tool/src/commands/customization/index.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license
+
+import { subCommand, makeCommandInfo } from "../../framework/command";
+
+export const commandInfo = makeCommandInfo("customization", "applies customizations to the SDK");
```

Comment:
Did we consider a name that allows for future additional commands? E.g. `dev-tool src merge`, `dev-tool src patch`, `dev-tool src analyze`, etc?

Reply:
This is a good point! I don't love that customization is too difficult for my fat fingers to type.

@willmtemple, @timovv, @dgetu - Do you have any ideas on a good name instead of `customization`?

Reply:
How about `tweaks` instead of customizations?

Reply:
`tweaks` sound like an interesting idea. Let's address this in a follow-up PR though.